python/langwatch/src/langwatch/telemetry/span.py

```
@@ -0,0 +1,826 @@
+            error=error,
+            timestamps=timestamps,
+        )
+
+    def evaluate(
+        self,
+        slug: str,
+        name: Optional[str] = None,
+        input: Optional[str] = None,
+        output: Optional[str] = None,
+        expected_output: Optional[str] = None,
+        contexts: Union[List[RAGChunk], List[str]] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)